libs/escrow-utils/src/constants.ts

```diff
@@ -152,6 +152,29 @@ export const NETWORK_CONFIG: { [key: number]: NetworkConfig } = {
       },
     },
   },
+  11155111: {
```

Comment:
hey can we pick this NETWORK_CONFIG directly from @smartinvoicexyz/constants ?

If there's anything extra that's specific to DM, we can modify the NETWORK_CONFIG or have a separate config here thats specific to DM.

Reply:
We could potentially get rid of escrow-utils package as well here unless its absolutely necessary

Reply:
@dan13ram Configurations are different between SI and DM - supported chains for example- that's why I exposed it in DM.
But we can change SI to make the properties of this object as optional and pass only the needed properties from NETWORK_CONFIG if that's better

Reply:
> We could potentially get rid of escrow-utils package as well here unless its absolutely necessary

Ok I will work on that.
But is it really better to make differsnt config and const just for DM inside SI?
Won't that change the package size for other users of the package?